scout/server/blueprints/variant/templates/variant/variant_details.html

```diff
@@ -195,29 +195,31 @@
 {% endmacro %}
 
 {% macro old_observations(variant, obs_date) %}
-  <div class="card panel-default">
-    <div class="panel-heading">
-      <a class="text-white" href="https://github.com/moonso/loqusdb" target="_blank">Local observations (archive {{obs_date or '2017-05-31'}})</a>
-    </div>
-    <div class="card-body">
-      <table class="table">
-        <thead class="thead-light">
-          <tr>
-            <th scope="col">Nr obs.</th>
-            <th scope="col">Nr homo.</th>
-            <th scope="col">Total nr.</th>
-          </tr>
-        </thead>
-        <tbody>
-          <tr>
-            <td>{{ variant.local_obs_old or 'N/A' }}</td>
-            <td>{{ variant.local_obs_hom_old or 'N/A' }}</td>
-            <td>{{ variant.local_obs_total_old or 638 }}</td>
-          </tr>
-        </tbody>
-      </table>
+  {% if config.SHOW_ARCHIVED_OBSERVATIONS is not defined or config.SHOW_ARCHIVED_OBSERVATIONS is true%}
```

Comment:
I haven't changed at all the code from this macro, I've just moved it inside the "IF"